src/config/topics.js

```diff
@@ -328,6 +332,52 @@ export const sts = {
   constrainOnlyCenter: true,
 };
 
+export const direktverbindungen = {
+  name: `${DV_KEY}.topic`,
+  key: `${DV_KEY}`,
+  elements: {
+    ...defaultElements,
+    popup: true,
+    shareMenu: true,
+    fitExtent: false,
+  },
+  maxZoom: 13,
+  minZoom: 6,
+  layers: dvLayers,
+  projection: 'EPSG:3857',
+  layerInfoComponent: 'DvTopicInfo',
+  searches: defaultSearches,
+  customMapControls: <DvListButton style={{ padding: 8, marginTop: 10 }} />,
```

Comment:
we don't use the word custom in similar property so call it 'mapControls'

Reply:
Done